keps/sig-scheduling/5075-dra-consumable-capacity/kep.yaml

```diff
@@ -0,0 +1,47 @@
+title: DRA: Consumable Capacity
+kep-number: 5075
+authors:
+  - "@sunya-ch"
+owning-sig: sig-scheduling
+participating-sigs:
+  - sig-node
+  - sig-network
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+creation-date: 2025-30-01
+reviewers:
+  - "@johnbelamaric"
+  - "@catblade"
+approvers:
+  - TBD
+
+see-also:
+#  - "/keps/sig-aaa/1234-we-heard-you-like-keps"
```

Comment:
Probably partitionable devices should be listed here and structured parameters.

Reply:
added 

```
see-also:
  - "/keps/sig-scheduling/4815-dra-partitionable-devices"
  - "/keps/sig-node/4381-dra-structured-parameters"
  - "/keps/sig-scheduling/5007-device-attach-before-pod-scheduled"
```